pkg/sources/git/git.go

```diff
@@ -616,3 +634,49 @@ func getSafeRemoteURL(repo *git.Repository, preferred string) string {
 	}
 	return safeURL
 }
+
+func handleBinary(repo *git.Repository, chunksChan chan (*sources.Chunk), chunkSkel *sources.Chunk, commitHash plumbing.Hash, path string) error {
```

Comment:
suggestion: unwrap the second arg type.
`func handleBinary(repo *git.Repository, chunksChan chan *sources.chunk, ...)`